invenio.cfg

```diff
@@ -341,3 +341,26 @@ RDM_CUSTOM_FIELDS = [
 # Search query of recent uploads
 # Defaults to newest records search
 ZENODO_FRONTPAGE_RECENT_UPLOADS_QUERY = "/api/records?sort=newest&size=10"
+
+
+# Journal CF
+
+from invenio_rdm_records.contrib.journal.custom_fields import (
+    JOURNAL_NAMESPACE,
+    JOURNAL_CUSTOM_FIELDS,
+    JOURNAL_CUSTOM_FIELDS_UI
+)
+
+RDM_CUSTOM_FIELDS_UI = [
+    JOURNAL_CUSTOM_FIELDS_UI
```

Comment:
is this an object or a list?

Reply:
`JOURNAL_CUSTOM_FIELDS_UI` is an object, while `RDM_CUSTOM_FIELDS_UI` is a [list](https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/config.py#L405)